OptimoveSDK/optimove-sdk/src/main/java/com/optimove/android/preferencecenter/OptimovePreferenceCenter.java

```diff
@@ -0,0 +1,281 @@
+package com.optimove.android.preferencecenter;
+
+import android.os.Handler;
+import android.os.Looper;
+import android.util.Log;
+
+import androidx.annotation.NonNull;
+import androidx.annotation.Nullable;
+
+import com.optimove.android.Optimove;
+import com.optimove.android.OptimoveConfig;
+import com.optimove.android.main.common.UserInfo;
+import com.optimove.android.main.tools.networking.HttpClient;
+
+import org.json.JSONArray;
+import org.json.JSONException;
+import org.json.JSONObject;
+
+import okhttp3.Response;
+
+import java.io.IOException;
+import java.net.URLEncoder;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Objects;
+import java.util.concurrent.ExecutorService;
+import java.util.concurrent.Executors;
+
+public class OptimovePreferenceCenter {
+    private static final String TAG = OptimovePreferenceCenter.class.getName();
```

Comment:
this TAG has no usefulness currently -- class names are obfuscated, so, it shows in logs as a.b.c.d . We could either 

a) add proguard rule
-keep class com.optimove.android.preferencecenter.** { *; }
-dontwarn com.optimove.android.preferencecenter.**

similar to how we do for everything under optimobile

b) just set TAG to "OptimovePreferenceCenter" 

think it's fine to obfuscate / leads to no issues here --> b) prob is better